src/pages/SearchPage/SearchPageFooter.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import {Info} from '@components/Icon/Expensicons';
+import {PressableWithoutFeedback} from '@components/Pressable';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function SearchPageFooter() {
+    const themeStyles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+
+    return (
+        <View style={[themeStyles.pb5, themeStyles.flexShrink0]}>
```

Comment:
The footer has a larger padding than what staging has

<img width="381" alt="image" src="https://github.com/Expensify/App/assets/29673073/ed3f46c9-4576-411f-b796-8c36dae59aa3">